packages/cli/src/commands/config/reset.ts

```diff
@@ -0,0 +1,31 @@
+import { CliUx } from '@oclif/core'
+import inquirer from 'inquirer'
+
+import { resetManagedConfig, SmartThingsCommand } from '@smartthings/cli-lib'
+
+
+export default class ConfigResetCommand extends SmartThingsCommand {
+	static description = 'clear saved answers to questions\n' +
+		'The CLI will occasionally ask you if you want it to remember the answer to a question, such as ' +
+		'"Which hub do you want to use?" You can use this command to clear those answers.'
+
+	static flags = SmartThingsCommand.flags
+
+	async run(): Promise<void> {
+		const { args, argv, flags } = await this.parse(ConfigResetCommand)
+		await super.setup(args, argv, flags)
```

Comment:
Can you explain what the benefit of this is? There doesn't seem to be any Oclif documentation for the `init` method, they don't use it in the Heroku CLI and it's not included when you use their CLI to generate a stub command. Our `run` functions are generally rather small anyway so removing these two lines and adding a whole function just seems to add more clutter to me.

Reply:
It would provide separation of concerns for initialization and command run logic. Utilizing OCLIFs init lifecycle in all our commands would remove the requirement for [base property access checks](https://github.com/SmartThingsCommunity/smartthings-cli/blob/4fb42dd5036bc2758cfb67d4f8cd24656142d9b6/packages/lib/src/smartthings-command.ts#L148), as we could be more confident that everything is initialized before run is called.

The usage of this in docs is sort of hidden in their [Custom Base Class](https://oclif.io/docs/base_class) page.